app/src/main/java/com/github/se/studybuddies/navigation/Route.kt

```diff
@@ -15,10 +15,11 @@ object Route {
   const val TIMER = "Timer"
   const val CALENDAR = "Calendar"
   const val CHAT = "GeneralChat"
+  const val CREATETODO = "CreateToDo"
+  const val EDITTODO = "EditToDo"
```

Comment:
You should add the ToDo list screen route here and not in navigation.

Reply:
it already exists there, under "TODOLIST", the navigation bar button was simply a placeholder since the solostudyscreen was not working